### PR TITLE
try to update recipe

### PR DESCRIPTION
**Pull request by wolfv**
locally (on osx-arm64) it fails with some issue about `panmirror`, which apparently is a project that now lives in quarto.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -37,125 +41,96 @@ requirements:
...
+    - nodejs >=18
     - {{ cdt('mesa-libgl-devel') }}  # [linux]
     - {{ cdt('mesa-dri-drivers') }}  # [linux]
     - {{ cdt('libselinux') }}  # [linux]
     - {{ cdt('libxdamage') }}  # [linux]
     - {{ cdt('libxxf86vm') }}  # [linux]
     - {{ cdt('libxext') }}     # [linux]
-
-outputs:
-  - name: rstudio-desktop
```
Comment (h-vetinari) [inline recipe/meta.yaml:49]: Please keep the outputs. Once we get `libpam` as a CDT we can add `rstudio-server` here.

File: recipe/meta.yaml [hunk 2]
```diff
@@ -37,125 +41,96 @@ requirements:
     - coreutils  # [osx]
     - patchelf  # [linux]
     - unzip
-    - nodejs
+    - nodejs >=18
```
Comment (h-vetinari) [inline recipe/meta.yaml:44]: Are we sure this is a good idea? nodejs has a per-major run-export, so building per version sounds much safer to me in terms of co-installability with other nodejs-using packages (that said, I'm not sure if https://github.com/conda-forge/rstudio-feedstock/pull/31/commits/96d47f1f668e6ee80698afb27e2f10a3e499faff was just intended as a debug commit to reduce the number of jobs)

**Review by h-vetinari — COMMENTED** [review]
Thanks so much for tackling this, and congrats on the first green build in a long time! 🥳

File: recipe/meta.yaml (hunk 2 above)
Comment (wolfv) [reply recipe/meta.yaml:44]: Nope. Then we should remove the run exports!

Comment (wolfv) [reply recipe/meta.yaml:44]: I mean - ignore the run exports here. We only need nodejs for building, right?

Comment (wolfv) [reply recipe/meta.yaml:44]: in fact, the run exports is probably `weak` so it shouldn't propagate to the run deps.

Comment (h-vetinari) [reply recipe/meta.yaml:44]: True, if it's a build-only dep it shouldn't matter. 👍

**Comment by wolfv** [comment]
It also helped to uncover a little bug in `rattler-build` :) https://github.com/prefix-dev/rattler-build/pull/781

**Comment by roaldarbol** [comment]
@izahn Can you give this a review? It seems to be ready to build. :-)

**Review by h-vetinari — CHANGES_REQUESTED** [review]
I'm grateful for the fixes, but this still hasn't addressed the review not to remove `outputs:`. I've cleaned up the changes from this PR into #32. If that passes, I'm happy to merge (unless @wolfv would like to clean up the commit history here to avoid the indentation churn).

**Comment by h-vetinari** [comment]
Superseded by #32

**Pull request closed without merging**